dumb-init.spec

```diff
@@ -0,0 +1,46 @@
+Name:           dumb-init
+Version:        1.1.3
+Release:        2%{?dist}
+Summary:        Entry-point for containers that proxies signals
+
+License:        MIT
+URL:            https://github.com/Yelp/dumb-init
+Source0:        https://github.com/Yelp/dumb-init/archive/v%{version}.tar.gz
+
+BuildRequires:  vim-common
```

Comment:
> still curious why this depends on vim-common? (GitHub buried my original comment, it seems)

because I have a line that runs `make VERSION.h` (although it's commented out) which calls /bin/xdd which belongs to vim-common

```
[root@localhost ~]# rpm -qf /bin/xxd
vim-common-7.4.1868-1.fc24.x86_64
```
You can make it depend on /bin/xxd as filename if you wish
You can comment both out  (so that the one who is building un-released version would enable both again)